pyanaconda/core/string.py

```diff
@@ -131,3 +131,25 @@ def have_word_match(str1, str2):
     str2 = str2.lower()
 
     return all(word in str2 for word in str1_words)
+
+
+def split_in_two(value, delimiter=None):
+    """Split the given value into two strings.
+
+    This function is useful for safe tuple unpacking.
+    The functionality is similar to str.partition(),
+    but it supports the delimiter of str.split().
+
+    For example:
+
+        first, second = split_in_two(text)
+
+    :param value: a string to split
+    :param delimiter: a delimiter for splitting
```

Comment:
I'm missing information what will happen in case of `None`. Could point to the documentation.

Reply:
+1, actually. I didn't like the magic `None` part, but given that it's a wrapper around `string.split`, I decided it's my fault I don't know the behavior well enough.

Reply:
Fixed.